tests/common/common.h

```diff
@@ -13,19 +13,19 @@
 #include "sycl.h"
 
 // test framework specific device selector
-#include "../common/cts_selector.h"
+#include "../../util/proxy.h"
 #include "../common/cts_async_handler.h"
+#include "../common/cts_selector.h"
 #include "../common/get_cts_object.h"
-#include "../../util/proxy.h"
 #include "macros.h"
 
```

Comment:
We should probably remove this empty line, so that clang-format will sort both groups together.

tests/common/common_vec.h has the same issue.

Reply:
Done. I've also removed the comment line in those two files regarding the device selector, which was misplaced due to the sorting and doesn't strike me as too important.

Reply:
Thanks!